src/NuGetGallery/Scripts/gallery/common.js

```diff
@@ -635,5 +635,17 @@
                 skippedToContent.focus();
             }
         });
+
+        window.WcpConsent && WcpConsent.init("en-US", "cookie-banner", function (err, _siteConsent) {
+            if (err !== undefined) {
+                console.log("Error initializing WcpConsent: " + err);
```

Comment:
Is `err` an object? If so, you can dump objects to the browser's console:

```suggestion
                console.log("Error initializing WcpConsent: ", err);
```

Please ignore this comment if `err` is a string.